.travis.yml

```diff
@@ -2,9 +2,12 @@ sudo: false
 
 language: go
 
-go:
-  - 1.4
-  - 1.3
+matrix:
+  include:
+    - go: 1.4
+      env: KUBE_TEST_API_VERSIONS="v1beta1"
+    - go: 1.3
+      env: KUBE_TEST_API_VERSIONS="v1beta3"
```

Comment:
For reference, here is the relevant documentation: http://docs.travis-ci.com/user/build-configuration/#Explicit-inclusion-of-jobs-into-the-build-matrix